packages/fx-core/src/common/m365/packageService.ts

```diff
@@ -90,32 +88,15 @@ export class PackageService {
         );
         const resCode = statusResponse.status;
         if (resCode === 200) {
-          complete = true;
+          const titleId: string = statusResponse.data.titleId;
+          const appId: string = statusResponse.data.appId;
+          this.logger?.info(`Acquire done. App TitleId: ${titleId}, AppId: ${appId}`);
```

Comment:
Office addin matches title id from the output, so need to keep the original titleid output format.
https://github.com/OfficeDev/Office-Addin-Scripts/blob/master/packages/office-addin-dev-settings/src/publish.ts#L19

Since there's no "Check acquired package" step, how about just output?
```
TitleId: ...
AppId: ...
Sideloading done.
```

Reply:
Updated.